logos/logos-workernode/tests/test_max_lanes.py

```
@@ -0,0 +1,151 @@
+"""Tests for the MAX_LANES feature (Feature 1).
+
+Validates that the LaneManager respects the max_lanes limit in add_lane(),
+apply_lanes(), and _add_lane_unlocked().
+"""
+
+from __future__ import annotations
+
+import asyncio
```

Flake8 (7.3.0)

[error] 'asyncio' imported but unused

(F401)